src/discordas.ts

```diff
@@ -94,8 +93,17 @@ async function run(port: number, fileConfig: DiscordBridgeConfig) {
                     return await roomhandler.OnAliasQuery.bind(roomhandler)(alias, aliasLocalpart);
                 } catch (err) { log.error("Exception thrown while handling \"onAliasQuery\" event", err); }
             },
-            onEvent: async (request, context) => {
+            onEvent: async (request) => {
                 try {
+                    // Build our own context.
```

Comment:
why do we need this?

Reply:
We used to use the nedb context in a few places to find out which remote room we were handling, now we do the same thing with the db.